bertopic/representation/_utils.py

```diff
@@ -44,8 +44,8 @@ def decode(self, doc_chunks):
         truncated_document: A truncated document
     """
     if doc_length is not None:
-        if tokenizer == "char":
-            truncated_document = document[:doc_length]
+        if tokenizer == "char" or tokenizer is None:
+            truncated_document = document[:doc_length] + " (...)"
```

Comment:
Could you explain why this was added? I believe something like this was addressed in https://github.com/MaartenGr/BERTopic/pull/2181

Reply:
Yes, some issue which was addressed in #2181